warehouse/query-core/src/test/java/datawave/query/RangeQueryTest.java

```diff
@@ -131,26 +139,28 @@ public void testSingleValueAndMultiFieldWithParens() throws Exception {
         }
     }
     
-    @Test
+    @Test(expected = FullTableScansDisallowedException.class)
```

Comment:
Previously this would pull together the bounds as a bounded range.  Now we required ranges to be bounded.  Hence this results only non-executable separate bounds and hence the query requires a full table scan required.